parser-typechecker/src/Unison/Util/Pretty.hs

```diff
@@ -786,6 +787,10 @@ blockedCallout = callout "🚫"
 backticked :: IsString s => Pretty s -> Pretty s
 backticked p = group ("`" <> p <> "`")
 
+-- |Attach some punctuation after the closing backtick.
+backticked' :: IsString s => Pretty s -> Pretty s -> Pretty s
+backticked' p end = group ("`" <> p <> "`" <> end)
+
```

Comment:
Side note, I would love a combinator that took a pretty, applied `P.wrap`, but then grouped a supplied punctuation or something to the last word of the `wrap`.